package.json

```diff
@@ -12,7 +12,9 @@
     "scripts": {
         "build": "rollup --config rollup.config.js",
         "watch": "rollup --config rollup.config.js -w",
-        "test": "node test/boilerplate/test-karma.js"
+        "test": "node test/boilerplate/test-karma.js",
```

Comment:
I know this is out of scope, but we should add a follow up item to remove the boilerplate tests in favor of the most recent test framework. The boilerplate framework is known to fail silently at times and I would rather we brought Braze in line with the rest of the kits.

Reply:
I'll add this work item to the rollup.test.config.js since I can't add comments to a json file.

https://go.mparticle.com/work/SQDSDKS-6875